### DISCUSSION TITLE
Weird sound behavior on Android phone with stereo speakers

### DISCUSSION THREAD

**Discussion by JacKeTUs** [ppy/osu#19699] [Q&A]
### Type

Other

### Bug description

I have one of these phones with front faced stereo speakers setup, where left channel is playing through amplified earpiece speaker

After launching osu!lazer i discovered that sound goes only from right speaker as mono sound. Left (earpiece) speaker is silent.
In other apps (Youtube, Poweramp) stereo sound is present, so its not a hardware issue with a phone

If i turn on Autorotate and rotate the phone at 180 degrees (now earpiece speaker on the right), i magically get that stereo sound from osu at every channel. 

Smartfone model: XQ-CT72, Android 12, version of ROM 64.0.A.8.25

### Screenshots or videos

https://user-images.githubusercontent.com/10255375/184050101-6897dc3c-5872-49ac-b1d5-52b1bf80945e.mp4

### Version

2022.810.2-lazer

### Logs

[runtime.log](https://github.com/ppy/osu/files/9305395/runtime.log)
[performance.log](https://github.com/ppy/osu/files/9305396/performance.log)
[network.log](https://github.com/ppy/osu/files/9305397/network.log)
[database.log](https://github.com/ppy/osu/files/9305398/database.log)

**Comment by peppy** [comment]
This sounds like a phone-level bug...

**Reply by JacKeTUs** [reply]
But Youtube and Poweramp (music player) is fine with every screen position (0, 90, 180, 270), thats what got me confused. I doubt that ROM has white-listed apps with white-listed screen positions for stereo...

I will try some other apps with this, and will give additional info if i can
